static/app/utils/fields/index.ts

```diff
@@ -1993,6 +1999,7 @@ export const ISSUE_EVENT_FIELDS_THAT_MAY_CONFLICT_WITH_TAGS: Set<FieldKey> = new
   FieldKey.STACK_MODULE,
   FieldKey.STACK_PACKAGE,
   FieldKey.STACK_STACK_LEVEL,
+  FieldKey.STATUS,
```

Comment:
@malwilley i wasn't totally sure about this change and how these are used already

Reply:
Yeah this is exactly what this was set was meant to solve so it belongs here